src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -56,19 +56,18 @@ case class LibrarySearchResponse(
 
 // classes to create the ES queries in json format
 // {"query":{"match_all":{}}}"
-// {"query":{"wildcard":{"_all":"%s"}}}"
+// {"query":{"match":{"_all":"<str>"}}}"
```

Comment:
it's not so useful in this version, but once we incorporate facets, it's helpful for me to be able to look at the example to understand the query generated instead of trying to re-work it mentally from the classes. i'm fine changing it to a different comment format (/**)